_docs/mapreduce.md

```diff
@@ -125,23 +125,24 @@ You'll start up one instance of splitter for each mapper, using a pipe to send `
 
 Command line:
 
-`./mapreduce <input_file> <output_file> <mapper_executable> <reducer_executable> <mapper_count>`
-
-Sample Usage
+`$ ./mapreduce <input_file> <output_file> <mapper_executable> <reducer_executable> <mapper_count>`
 
+Sample Usage:
 
 `$ ./mapreduce test.in test.out ./my_mapper ./my_reducer 3`
 
 
 Your program will:
 
-* Split the input file into <mapper_count> parts and pipe the contents into <mapper_count> different mapper processes (use splitter).
+* Split the input file into \<mapper_count\> parts and pipe the contents into \<mapper_count\> different mapper processes (use splitter).
```

Comment:
Suggestion: use teletype for `mapper_count`

Reply:
Teletype?

Reply:
Basically, put it as `mapper_count` instead of mapper_count

Reply:
Can do